src/include/hardware.h

```diff
@@ -16,6 +16,32 @@
         (v) = (val);        \
     } while (0)
 
+/*
+ * WT_READ_ONCE --
+ *
+ * Ensure a single read from memory in the source code produces a single read from memory in the
+ * compiled output.
+ *
+ * The compiler is allowed to optimize loads from memory in multiple ways such as Load fusing, where
```

Comment:
I think using quotes here to designate naming a thing would be better. So such as "load fusing" and "invented loads".

Reply:
Added in [c460ce8](https://github.com/wiredtiger/wiredtiger/pull/9413/commits/c460ce81236e86dbce5ca0e35af8610fbd0a4260)